.github/workflows/ci.yml

```diff
@@ -288,127 +288,50 @@ jobs:
-        ENV_NETWORK_NAME="gma-a-df-vpc"
-        ENV_SUBNETWORK_NAME="gma-a-df-central1"
-        ENV_TEMPLATE_FILE="./config/commit-review-status-metadata.json"
-        ENV_PROJECT_ID="${{ env.AUTOPUSH_PROJECT_ID }}"
-        ENV_DATASET="github_metrics"
-        
-        BUCKET_NAME="gs://${ENV_BUCKET_NAME}"
-        TEMPLATE_FILE="${BUCKET_NAME}/templates/commit-review-status.json"
-        
-        CONTAINER="${{ env.DOCKER_REPO }}/commit-review-status-pipeline:${{ env.DOCKER_TAG }}-amd64"
-        
-        GH_PK_SECRET="${{ secrets.GH_PRIVATE_KEY_SECRET }}"
-        GH_APP_ID="${{ secrets.GH_APP_ID }}"
-        GH_INSTALL_ID="${{ secrets.GH_INSTALL_ID }}"
-        
-        PUSH_EVENTS_TABLE="${ENV_PROJECT_ID}.${ENV_DATASET}.push_events"
-        COMMIT_REVIEW_STATUS_TABLE="${ENV_PROJECT_ID}.${ENV_DATASET}.commit_review_status"
-        ISSUES_TABLE="${ENV_PROJECT_ID}.${ENV_DATASET}.issues"
-        
-        gcloud dataflow flex-template build "${TEMPLATE_FILE}" --image "${CONTAINER}" --sdk-language "GO" --metadata-file "${ENV_TEMPLATE_FILE}"
-        
-        gcloud beta datapipelines pipeline delete "commit-review-status" --project="${ENV_PROJECT_ID}" --region "${ENV_REGION}"
-        
-        gcloud beta datapipelines pipeline create "commit-review-status" \
-            --project="${ENV_PROJECT_ID}" \
-            --region="${ENV_REGION}" \
-            --pipeline-type="BATCH" \
-            --template-file-gcs-location="${TEMPLATE_FILE}" \
-            --network "${ENV_NETWORK_NAME}" \
-            --subnetwork "https://www.googleapis.com/compute/v1/projects/${ENV_PROJECT_ID}/regions/${ENV_REGION}/subnetworks/${ENV_SUBNETWORK_NAME}" \
-            --schedule="0 */4 * * *" \
-            --worker-region="${ENV_REGION}" \
-            --disable-public-ips \
-            --additional-experiments=use_prime \
-            --parameters=batch-size=github-app-id="${GH_APP_ID}",github-app-installation-id="${GH_INSTALL_ID}",github-app-private-key-resource-name="${GH_PK_SECRET}",push-events-table="${PUSH_EVENTS_TABLE}",commit-review-status-table="${COMMIT_REVIEW_STATUS_TABLE}",issues-table="${ISSUES_TABLE}"
+      - name: 'Deploy Commit Review Status Pipeline'
+        uses: 'abcxyz/pkg/.github/actions/deploy-dataflow-pipeline@main' # ratchet:exclude
+        with:
+          wif_provider: '${{ vars.WIF_PROVIDER }}'
+          wif_service_account: '${{ vars.WIF_SERVICE_ACCOUNT }}'
+          image: '${{ env.DOCKER_REPO }}/commit-review-status-pipeline:${{ env.DOCKER_TAG }}-amd64'
+          template_metadata_file: './config/commit-review-status-metadata.json'
+          template_file_name: 'commit-review-status.json'
+          project: '${{ env.AUTOPUSH_PROJECT_ID }}'
+          region: '${{ env.AUTOPUSH_REGION }}'
+          bucket_name: 'github-metrics-aggreg-a-997e5e-df-store'
+          network_name: 'gma-a-df-vpc'
+          subnet_name: 'gma-a-df-central1'
+          pipeline_name: 'commit-review-status'
```

Comment:
nit: the other pipeline name is prefixed with `gma-`, can we do the same here?

Reply:
done.